src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx

```diff
@@ -220,7 +222,13 @@ const ProjectSelector = createReactClass({
 
   getProjectLabel(team, project, hasSingleTeam, highlightText) {
     let label, text;
-    if (!hasSingleTeam && project.name.indexOf(team.name) === -1) {
+    let features = new Set(this.props.organization.features);
+
+    if (
+      !hasSingleTeam &&
+      project.name.indexOf(team.name) === -1 &&
+      !features.has('internal-catchall')
```

Comment:
One question - in the new, internal behavior, do we want to show `slug` or `name`?

Reply:
mm i remember that conversation happening and do not remember what the decision was :-/

Reply:
ok we're keeping name